Index: src/test/java/de/dfki/lt/tools/tokenizer/annotate/TestFastAnnotatedString.java
===================================================================
--- src/test/java/de/dfki/lt/tools/tokenizer/annotate/TestFastAnnotatedString.java	(revision 236)
+++ src/test/java/de/dfki/lt/tools/tokenizer/annotate/TestFastAnnotatedString.java	(working copy)
@@ -62,14 +62,14 @@
     input1.annotate("type", "tok", 8, 9);
     input1.annotate("type", "tok", 10, 14);
     input1.annotate("type", "punct", 14, 15);
-    compareResults(input1, "expected-results/annotated-String-expected-1.txt");
+    compareResults(input1, "expected-results/annotated-string-expected-1.txt");
 
     AnnotatedString input2 = new FastAnnotatedString("sdfslkdflsdfsldfksdf");
     input2.annotate("type", "tok", 5, 15);
     assertEquals("kdflsdfsld\t5-15\ttok", input2.toString("type").trim());
 
     input2.annotate("type", "mid", 9, 12);
-    compareResults(input2, "expected-results/annotated-String-expected-2.txt");
+    compareResults(input2, "expected-results/annotated-string-expected-2.txt");
   }
 
